tests/models/abiotic/test_conductivities.py

```diff
@@ -23,46 +23,20 @@ def test_initialise_conductivities(dummy_climate_data, fixture_core_components):
         bottom_leaf_air_conductivity=0.13,
     )
 
-    coords = {
-        "layers": np.arange(15),
-        "layer_roles": (
-            "layers",
-            fixture_core_components.layer_structure.layer_roles,
-        ),
-        "cell_id": [0, 1, 2],
-    }
+    exp_air_cond = fixture_core_components.layer_structure.from_template()
+    # VIVI - these test values have changed from:
```

Comment:
yes, that results from a different number of layer heights

Reply:
I think the function assumes that the atmosphere layers all have the same depth which is not true, I'll add a TODO to the function.